pkgs/development/python-modules/pymusic-dl/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, click
+, prettytable
+, pycryptodome
+, pytest
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "pymusic-dl";
+  version = "3.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-A/cBv7TIZX3Vhua3HNwV5sbQmk/DrePm9qykV9X9Rko=";
+  };
+
+  propagatedBuildInputs = [
+    click
+    prettytable
+    pycryptodome
+    requests
+  ];
+
+  checkInputs = [
+    pytest
```

Comment:
Could you check if `pytestCheckHook` could be used.

Reply:
Yes. `pytestCheckHook` works.